users/tests/test_api_user.py

```diff
@@ -0,0 +1,222 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse
+from rest_framework.exceptions import ErrorDetail
+from rest_framework.test import APIClient
+
+User = get_user_model()
+
+
+@pytest.fixture()
+def api_user_url(user: User) -> str:
+    return reverse("api-user", kwargs={"nickname": user.nickname})
+
+
+@pytest.fixture()
+def api_user_profile_url() -> str:
+    return reverse("api-user-profile")
+
+
+@pytest.fixture()
+def api_user_settings_url() -> str:
+    return reverse("api-user-settings")
+
+
+@pytest.mark.django_db()
+def test_api_user_valid_data_response_success(
```

Comment:
A bit of naming issue. 
The tests is checking GET method. But the name is confusing and for me suggests it's updae. 
Please change the name of each of the test to clearly state is it get/patch/post(create) or other. 
THe problem is I need to read carefully test code to see it GET. So when I am seeking for the test to GET, I can't use collapsed view, I need to have all the code listed and read it through, which is a waste of time. 
Thank you.